### DISCUSSION TITLE
How to check a field has attachment?

### DISCUSSION THREAD

**Discussion by alwaysq** [pocketbase/pocketbase#4757] [Q&A]
I searched this discussion page to find how to search a record with attachement, but couldn't. The codes below does not work.   Could you let me know how can I do it?

=============================
this.items = await this.client.collection("item").getFullList({
    filter: "picture != NULL", // when the 'picture' ficture is not empty? That is when 'picture' field has at least one attachement?
  });

Pocketbase is one of the best applications I have ever experienced as a newbie. Thank you always. :)

**Comment by ganigeorgiev** [comment]
`picture != null` or `picture != ""` should work for single `file` field.
For multiple `file` field the empty value is a serialized json array, so you'll need something like `picture != "[]"`

An alternative could be to use the [`:length` modifier](https://pocketbase.io/docs/api-rules-and-filters/#length-modifier) as this will work for both single and multiple fields:

```js
picture:length > 0
```

If that doesn't help, please export your collections configuration as json from the _Admin UI > Settings > Export collections_.

**Reply by alwaysq** [reply]
Thank you so much. It worked like a charm. (picture:length > 0).
Learning how to use Pocketbase is a job of my life :)
